README.md

```diff
@@ -11,6 +11,33 @@ a prototype of a _**Keptn Lifecycle Controller**_.
 The goal of this prototype is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.
 It is an experimental project, under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+## Keptn Lifecycle Controller in a nutshell
+
+The idea for **Keptn Lifecycle Controller** is to allow managing the App Lifecycle without writing operator code.
+This approach is also known as a _Level 3 Operator_
+It targets cloud native applications deployed in Kubernetes and managed via modern approaches like GitOps.
+It is not yet another deployment tool but an add-on that can easily integrate into your application management and CD ecosystem
+to enable cloud native app operations.
+
+Keptn Lifecycle Controller:
+
+- Provides a standardized way to ensure that applications are running perfectly:
+  Pre-Deployment, Post-Deployment, and Application Health Checks.
+- Supports connecting Event Hooks to other stages of the application lifecycle.
```

Comment:
This is a bit unclear to me ... What are Event Hooks in KLC?

Reply:
It comes from older slides.
How about: "Connects to other tools via CRDs or webhooks"?